components/camel-corda/pom.xml

```diff
@@ -36,6 +36,20 @@
     <properties>
     </properties>
 
+    <repositories>
```

Comment:
Please don't use third party repository. It's a nightmare. We already passed through this with restlet. If it's not available in central we'll wait for it. But please avoid third party repository.

Reply:
I have already raised with R3 to release the corda-rpc:4.1 to maven central ... Hope this will be fixed soon

Reply:
@oscerd , I have not tested karaf feature .. Would you be able to guid me how to perform karaf feature?

Reply:
Thanks @oscerd .. Thats done .. All good